Orange/widgets/regression/owsgdregression.py

```diff
@@ -115,7 +114,10 @@ def __init__(self):
         power_t = gui.doubleSpin(box, self, "power_t", 0.0, 10.0, step=0.01)
         form.addRow("Power t:", power_t)
 
-
+        scroll = QtGui.QScrollArea()
+        scroll.setWidget(box)
+        scroll.setWidgetResizable(True)
+        scroll.setFixedHeight(400)
```

Comment:
Ah, interesting. What you are saying is that sometimes the control area of widgets is too large (too high) and it doesn't fit in the screen's vertical estate. I totally agree and I hate it when it happens. Wrapping control area contents inside a `QScrollArea` should alleviate this, but it should probably be done once on `OWWidget` for all widgets that extend it.